src/shared/cpu-set-util.c

```diff
@@ -206,3 +206,42 @@ int cpus_in_affinity_mask(void) {
                 n *= 2;
         }
 }
+
+int cpu_set_to_dbus(const CPUSet *set, uint8_t **ret, size_t *allocated) {
+        uint8_t *out;
+
+        assert(set);
+        assert(ret);
+
+        out = new0(uint8_t, set->allocated);
+        if (!out)
+                return -ENOMEM;
+
+        for (unsigned cpu = 0; cpu < set->allocated * 8; cpu++)
```

Comment:
unsigned → size_t?

Reply:
It's used in the glibc API which only takes unsigned ...